jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  setupFilesAfterEnv: [
+    'jest-extended',
+  ],
```

Comment:
Try enabling in the test file itself (like https://github.com/thewilkybarkid/article-store/blob/d0a1f2d1adfe67f5f665af36e5f3603e54351a1f/test/middleware/error-handler.test.ts#L2).

Reply:
Did the trick!